ReadModel/src/ModelGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using EventReader.Event;
+using EventReader.Read;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ReadModel
+{
+    public class ModelGenerator
+    {
+        EventStream Stream;
+        public Dictionary<Guid, Customer> Customers { get; }
+
+        const string CUSTOMER_QUERY =
```

Comment:
We've got SQL way up in our model - the model should have no dependencies on the underlying mechanism for retrieving events.  Depend only on abstractions - we have a nice abstraction for a stream of events, namely `IEnumerable<IEvent>`